man/mprof-report.1

```diff
@@ -180,9 +180,17 @@ with the current process id and \f[I]%t\f[] with the current date
 and time, then do according to \f[I]OUTSPEC\f[]:
 .RS 2
 .IP \[bu] 2
+If \f[I]OUTSPEC\f[] begins with a \f[I]+\f[] character, then the
```

Comment:
The plan is to remove this option exactly because `output=stuff.mlpd.%p` does the same thing. So I'd rather not document it.